examples/webauthn/index.js

```diff
@@ -12,12 +12,12 @@ const XPUBS = 1;
 const ADDRESSES = 4;
 const CHANGE = true;
 
-(async () => {
-  const device = new Device({
-    scrambleKey: 'btc',
-    timeout: 20000
-  });
+const device = new Device({
+  scrambleKey: 'btc',
```

Comment:
I think you should delete `scrambleKey` from the example because its generally something that users don't need to configure since its set to the default value by default

Reply:
lol, true they might think this is different.
In theory, this should be compatible with some coins (that are built on top of the btc app) but we don't support them, so makes sense to get rid of this from examples.